config/routes.rb

```diff
@@ -20,13 +20,12 @@ def matches?(request)
     end
 
     constraints account: /[^\/\?]+/ do
-      get  '/authn/:account/login' => 'credentials#login'
-      put  '/authn/:account/password' => 'credentials#update_password'
-      put  '/authn/:account/api_key'  => 'credentials#rotate_api_key'
```

Comment:
From John in slack:
> why are we moving `update_password` and `rotate_api_key`? is it because you can perform those actions through the UI and we also want to be able to use LDAP creds?

https://conjurhq.slack.com/archives/CA8AT283D/p1537368039000100

Reply:
>Those only require the login "level" of authentication to perform. But with this, login isn't its own special case anymore, so they need to be authenticator aware. These also don't make sense outside of built-in authenticator though, so I do need to revisit that again.